index.js

```diff
@@ -32,12 +33,34 @@ module.exports = {
     'guard-for-in': 2,
     'indent': [2, 2, { 'SwitchCase': 1 }],
     'jsx-quotes': [2, 'prefer-single'],
-    'import/prefer-default-export': 1,
-    'import/named': 1,
-    'import/newline-after-import': 1,
-    'import/no-duplicates': 1,
-    'import/no-mutable-exports': 1,
-    'import/imports-first': 1,
+    'import/default': 0, // TODO: enable once module.exports is supported
+    'import/extensions': 2,
+    'import/first': 2,
+    'import/newline-after-import': 2,
+    'import/no-absolute-path': 2,
+    'import/no-anonymous-default-export': 1, // TODO: discuss if this should be a 2
+    'import/no-deprecated': 1, // NOTE: this rule is still in WIP stage
+    'import/no-duplicates': 2,
+    'import/no-dynamic-require': 2,
+    'import/no-extraneous-dependencies': 2,
+    'import/no-internal-modules': 2,
+    'import/no-mutable-exports': 2,
+    'import/no-named-as-default-member': 2,
+    'import/no-named-as-default': 2,
+    'import/no-named-default': 2,
+    'import/no-namespace': 2,
+    'import/no-unassigned-import': 2,
+    'import/order': [2, {
+      groups: [
+        'builtin',
+        'external',
+        'internal',
+        'index',
```

Comment:
@trotzig I'd rather have a discussion on what the order should be; `import-js` is not a tool that users of this library are expected to have/use. (Can you expand on what you mean when you say it will work in most cases? Does `import-js` read the values of this rule?

If we don't specify all the groups, the behavior is:

> [Omitted types are implicitly grouped together as the last element.](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array)

I disagree with that. I strongly believe that builtin imports, as well as external and internal package imports, should come before the `index`, `sibling`, and `parent` imports.